src/openzaak/components/catalogi/admin/utils.py

```diff
@@ -220,6 +226,10 @@ def import_zaaktype_for_catalogus(
                                 "A validation error occurred while deserializing a {}\n{}"
                             ).format(resource, deserialized.errors)
                         )
+            else:
+                files_not_found.append(f"{resource}.json")
+
+    return files_found, files_not_found, files_received
```

Comment:
Please add a typehint and perhaps a docstring to describe what is being returned in the function

Reply:
I  just removed the list returns and throw and catch an exception instead.